htdocs/core/class/extrafields.class.php

```diff
@@ -1266,11 +1266,10 @@ function showOutputField($key,$value,$moreparam='')
 		}
 		elseif ($type == 'boolean')
 		{
-			$checked='';
-			if (!empty($value)) {
-				$checked=' checked ';
-			}
-			$value='<input type="checkbox" '.$checked.' '.($moreparam?$moreparam:'').' readonly disabled>';
+			if (!empty($value)) 
```

Comment:
on/off may be  a label not appropriate for a lot of case.
I recommand to introduce a new type "onoff" instead of "boolean".